### DISCUSSION TITLE
Where should the colors&styles added in App.xml be added now?

### DISCUSSION THREAD

**Discussion by CeSun** [Dreamescaper/BlazorBindings.Maui#96] [General]
![image](https://user-images.githubusercontent.com/54771283/211186831-5bf1365c-66f9-41c4-8084-341396206ab4.png)

**Comment by Dreamescaper** [comment]
In this case - you can simply add it to your App.cs file:
```csharp
    public partial class App : Application
    {
        public App(MauiBlazorBindingsRenderer renderer)
        {
            Resources.Add(new Material.Components.Maui.Styles.MaterialStyles());

            renderer.AddComponent<AppShell>(this);
        }
    }
```

**Reply by CeSun** [reply]
> In this case - you can simply add it to your App.cs file:
> ```csharp
>     public partial class App : Application
>     {
>         public App(MauiBlazorBindingsRenderer renderer)
>         {
>             Resources.Add(new Material.Components.Maui.Styles.MaterialStyles());
> 
>             renderer.AddComponent<AppShell>(this);
>         }
>     }
> ```
It's worked, Thank you😁.
